test/jasmine/tests/gl_plot_interact_test.js

```diff
@@ -761,20 +763,18 @@ describe('gl2d interaction', function() {
     it('data-referenced annotations should update on drag', function(done) {
 
         function drag(start, end) {
-            var opts = { buttons: 1 };
-
-            mouseEvent('mousemove', start[0], start[1], opts);
-            mouseEvent('mousedown', start[0], start[1], opts);
-            mouseEvent('mousemove', end[0], end[1], opts);
-            mouseEvent('mouseup', end[0], end[1], opts);
+            mouseEvent('mousemove', start[0], start[1]);
+            mouseEvent('mousedown', start[0], start[1], { buttons: 1 });
+            mouseEvent('mousemove', end[0], end[1], { buttons: 1 });
+            mouseEvent('mouseup', end[0], end[1]);
```

Comment:
Full disclosure, I actually noticed this broken test case twice when releasing `1.22.0` and `1.23.0`, but I was lazy and didn't feel like making a PR _just for that_.